src/main/java/mate/academy/bookshop/model/Book.java

```diff
@@ -9,10 +9,14 @@
 import java.math.BigDecimal;
 import lombok.Getter;
 import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
 
 @Entity
 @Setter
 @Getter
+@SQLDelete(sql = "UPDATE books SET is_deleted = true WHERE id=?")
+@Where(clause = "is_deleted=false")
```

Comment:
@Where is Deprecated, use this instead.
```suggestion
@SQLRestriction("is_deleted=false")
```

Reply:
++